include/sample_rate_converter.h

```diff
@@ -39,7 +39,8 @@
 
 /** Filter types supported by the sample rate converter */
 enum sample_rate_converter_filter {
-	SAMPLE_RATE_FILTER_TEST = 1
+	SAMPLE_RATE_FILTER_TEST = 1,
+	SAMPLE_RATE_FILTER_SMALL
```

Comment:
Consideration: Can/should these be named "simple"? Since they reduce both size and time, "small" is not that descriptive.

Reply:
Good point, agreed!